src/test/java/redis/clients/jedis/tests/commands/ObjectCommandsTest.java

```diff
@@ -45,4 +49,27 @@ public void objectIdletime() throws InterruptedException {
     time = jedis.objectIdletime(binaryKey);
     assertEquals(new Long(0), time);
   }
+
+  @Test
+  public void objectHelp() {
+    List<String> helpTexts = jedis.objectHelp();
+    assertNotNull(helpTexts);
+  }
+
+  @Test
+  public void objectFreq() {
+    jedis.set(key, "test1");
+    // Before we test objectFreq command, we must config maxmemory-policy or will throw "An LFU maxmemory policy is not selected, access frequency not tracked. Please note that when switching between policies at runtime LRU and LFU data will take some time to adjust."
+    jedis.configSet("maxmemory-policy", "allkeys-lfu");
```

Comment:
It'd better to do this through config file (Makefile). You may either create a new Redis instance or modify+reuse a *safe* instance, perhaps REDIS8_CONF at port 6386.

Reply:
I reuse REDIS8_CONF at port 6386 and modify the config